hooks/post_gen_project.py

```diff
@@ -324,6 +312,19 @@ def remove_storages_module():
     os.remove(os.path.join("{{cookiecutter.project_slug}}", "utils", "storages.py"))
 
 
+def create_webpack_project():
+    from cookiecutter.main import cookiecutter
+
+    cookiecutter(
+        "https://github.com/AccordBox/python-webpack-boilerplate",
+        directory="frontend_template",
+        no_input=True,
+    )
```

Comment:
This is slightly concerning. Although less maintenance for @browniebroke is great, coming from a Swift background (i.e. SPM), I have a really hard time trusting dependencies solely based on a GitHub URL. But I guess this is up to the maintainers.

Reply:
Yes, I share the concern regarding trust... On the maintenance side of things, it's not clear win either: on one hand it's sort of outsourcing this part to an external package, but on the other hand I'm wondering how we can safely manage breaking changes. If this other template changes something we don't expect, we would break, causing maintenance issues. 

We need to make sure `python-webpack-boilerplate` and this line are using the same version, and we need to update this line each time we receive an automated updated for `python-webpack-boilerplate` from PyUP.

Calling `cookiecutter` programmatically is a really interesting idea, but I'm not sure we can consider it unless we own both projects. Right now, this part is a black box to me, and if anything were to go wrong, I would have a hard time debugging it.

Reply:
@Andrew-Chen-Wang @browniebroke 

Well, I think this is the biggest issue in this PR.

My solution is like this:

1. In the `hooks/post_gen_project.py`, I extract version of `python-webpack-boilerplate` from the `requirements/base.txt`
1. If the version is `0.1.2`, then `cookiecutter` the frontend project from Github tag url https://github.com/AccordBox/python-webpack-boilerplate/tree/v2.12.3

This can make sure same version are always used and can also help developers troubleshoot some potential issues.

Do you think is this good to go?